src/backend/marsha/core/api.py

```diff
@@ -497,25 +498,39 @@ def update_live_state(
         if not validate_signature(request.headers.get("X-Marsha-Signature"), msg):
             return Response("Forbidden", status=403)
 
-        video = self.get_object()
-        video.live_state = serializer.validated_data["state"]
-        live_info = video.live_info
-        live_info.update(
-            {"cloudwatch": {"logGroupName": serializer.validated_data["logGroupName"]}}
-        )
-
-        if video.live_state == defaults.RUNNING:
-            live_info.update({"started_at": stamp})
-            video.live_info = live_info
-
-        if video.live_state == defaults.STOPPED:
-            live_info.update({"stopped_at": stamp})
-            video.upload_state = defaults.HARVESTING
-            video.live_info = live_info
-            create_mediapackage_harvest_job(video)
-            delete_aws_element_stack(video)
+        with transaction.atomic():
+            try:
+                video = Video.objects.select_for_update(nowait=True).get(pk=pk)
+            except Video.DoesNotExist:
+                return Response({"success": False}, status=404)
+            except DatabaseError:
+                return Response({"success": True})
```

Comment:
Returning 200 when a DatabaseError occurs is a bit surprising, maybe add a comment ?